plugins/uefi-dbx/dbxtool.md

```diff
@@ -1,35 +1,40 @@
+---
+title: dbxtool command line utility
+---
+
 % dbxtool(1) {{PACKAGE_VERSION}} | dbxtool man page
 
 NAME
-====
+----
 
 **dbxtool** — modify the dbx revocation list
 
 SYNOPSIS
-========
+----
```

Comment:
I've got no problem with this conceptually -- but doesn't the old one look better?

Reply:
Doesn't it look the same in the result? The important part is consistency across files. Otherwise we'll need to modify the helper converter script to cover more cases.

Reply:
I think the markdown purists (of which there are certainly lots in this world!) would get quite upset at the `----` one.

Reply:
Adjusted to match the "regular" syntax specified in https://www.markdownguide.org/basic-syntax/#:~:text=Headings%20%20%20%20Markdown%20%20%20,Heading%20level%204%20%202%20more%20rows%20